inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp

```diff
@@ -2,130 +2,189 @@
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
         std::tie(postOpMgrPtr, fusedOps) = fusingParams;
+        if (selectedType.empty()) {
+            selectedType = getPrimitiveType();
+        }
+        selectedType = makeSelectedTypeStr("unknown", inType);
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        init_input_shapes({shapes});
 
-        std::vector<int64_t> axes;
-        float eps;
-        op::EpsMode eps_mode;
-        SizeVector inputShapes;
-        std::tie(axes, eps, eps_mode, inputShapes, inPrc, targetDevice) = basicParamsSet;
-
-        outPrc = inPrc;
-        auto netPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inPrc);
-        auto params = builder::makeParams(netPrc, {inputShapes});
-        auto paramOuts = helpers::convert2OutputVector(helpers::castOps2Nodes<op::Parameter>(params));
-        auto normalize = builder::makeNormalizeL2(paramOuts[0], axes, eps, eps_mode);
-
-        function = makeNgraphFunction(netPrc, params, normalize, "Normalize");
+        auto params = ngraph::builder::makeDynamicParams(inType, inputDynamicShapes);
+        auto normalize = builder::makeNormalizeL2(params[0], axes, eps, epsMode);
+        function = makeNgraphFunction(inType, params, normalize, "Normalize");
+    }
 
-        selectedType = "unknown_" + std::string(inPrc.name());
-        threshold = 0.015f;
+    void generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) override {
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (int i = 0; i < funcInputs.size(); ++i) {
+            const auto& funcInput = funcInputs[i];
+            ov::runtime::Tensor tensor;
+            if (funcInput.get_element_type().is_real()) {
+                tensor = ov::test::utils::create_and_fill_tensor(
+                        funcInput.get_element_type(), targetInputStaticShapes[i], 10, -5, 7, 222);
+            } else {
+                tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i]);
+            }
+            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
+        }
     }
 };
 
 TEST_P(NormalizeL2LayerCPUTest, CompareWithRefs) {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
 
-    Run();
+    run();
+
     CheckPluginRelatedResults(executableNetwork, "NormalizeL2");
 }
 
 namespace {
 
 /* ============= Common params ============= */
 std::vector<fusingSpecificParams> fusingParamsSet {
-        emptyFusingSpec,
-        fusingMultiplyPerTensor,
-        fusingMultiplyPerChannel,
-        fusingAddPerTensor,
-        fusingAddPerChannel,
-        fusingSubtractPerTensor,
-        fusingSubtractPerChannel,
-        fusingDividePerTensor,
-        fusingDividePerChannel,
-        fusingPReluPerChannel,
-        fusingPReluPerTensor,
-        fusingRelu,
-        fusingGelu,
-        fusingReluScaleShift
+    emptyFusingSpec,
+    fusingMultiplyPerTensor,
```

Comment:
I guess we need to reduce number of tested post

Please add several test cases with per channel post ops.
You need to create dynamic shapes with static channel for this.

Reply:
Added, thanks